draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -564,20 +564,34 @@
             </t>
             <t>if the Responder chooses to send Cookie request (possibly along
             with Puzzle request), then the TCP connection that the IKE_SA_INIT
-            request message was received over SHOULD be closed, so that the
-            Responder remains stateless at least until the Cookie (or Puzzle
-            Solution) is returned. Note that if this TCP connection is closed,
-			the Responder MUST NOT include the Initiator's TCP port into the Cookie
-			calculation (*), since the Cookie will be returned over a new
+            request message was received over SHOULD be closed after the Responder sends its reply
+            and no repeated requests are received within some short period of time (*)
```

Comment:
I find these (*) references hard to follow, particularly with large amounts of text. Why not just have a sub-section with a proper section reference?

Reply:
I created a new subsection discussing selecting time interval and referenced it in the text. The text about TCP ports in Cookie calculation is left under (*). Please review.